### DISCUSSION TITLE
Support for censored probabilistic regression (survival analysis; AFT models)

### DISCUSSION THREAD

**Discussion by gmgeorg** [StatMixedML/XGBoostLSS#53] [Ideas]
Thanks for this great library!

I was wondering if you plan to include censored loss functions as well to allow for censored probabilistic regression. Similar to what XGB AFT models do, but with a non-constant scale parameters in the distribution models (https://xgboost.readthedocs.io/en/stable/tutorials/aft_survival_analysis.html).

**Comment by StatMixedML** [comment]
@gmgeorg Thanks for your interest in the project. Great suggestion, I`ll add it to my ToDo.

**Reply by John-Curcio** [reply]
[This PR](https://github.com/StatMixedML/XGBoostLSS/pull/91) attempts to check that off the list
